src/LttngHelpers.c

```diff
@@ -351,7 +437,8 @@ static void *FixArray(
         {
             if (*ppGood == *(ppGood + 1))
             {
-                for (void const **ppNext = ppGood + 2; ppNext != ppEnd; ppNext += 1)
+                void const** ppNext;
```

Comment:
Intellisense was giving me grief about declaring the variables inside the `for` statement, so I moved them out just to make Intellisense happy.